### ISSUE TITLE
emitWarning option dont work

### ISSUE THREAD

**Issue by z-950** [webpack-contrib/eslint-webpack-plugin#19]
- Operating System: Windows 10 v2004 64-bit
- Node Version: 12.16.3
- NPM Version: 6.14.4
- webpack Version: 4.44.2
- eslint-webpack-plugin Version: 2.1.0

### Expected Behavior

Warning message should not be shown when emitWarning set to false.

### Actual Behavior

Warning message is shown when emitWarning set to false.

### Code

```js
// webpack.config.js
const path = require('path')
const ESLintPlugin = require('eslint-webpack-plugin')

module.exports = {
  mode: "development",
  entry: 'src/index.js',
  output: {
    path: path.resolve(__dirname, 'dist'),
    filename: 'index.js',
  },
  plugins: [new ESLintPlugin({
    context: 'src',
    extensions: ['js'],
    emitWarning: false,
    emitError: true,
    failOnError: true,
    failOnWarning: false
  })]
}
```
In this case, warning is always shown.

I use webpack-cli@3.3.12  to run building.

### How Do We Reproduce?

Warning message should not be shown when emitWarning set to false.

**Comment by cadenzah** [comment]
Seems like it's still not fixed. `quiet` option does not work either.
